docs/User_Quickstart.md

```diff
@@ -0,0 +1,31 @@
+---
+id: User_Quickstart
+title: Arbitrum User Quickstart
+sidebar_label: User Quickstart
+custom_edit_url:https://github.com/OffchainLabs/arbitrum/edit/master/docs/Developer_Quickstart.md
+---
+
+Arbitrum is a suite of Ethereum scaling solutions that enables high-throughput, low cost smart contracts while remaining trustlessly secure. Arbitrum has three modes: AnyTrust Channels, AnyTrust Sidechains, and Arbitrum Rollup. The following documentation describes how to use Arbitrum Rollup, which is currently live on testnet.
```

Comment:
Do we even want to mention the other two modes at this stage (/generally)? Pinging @edfelten @hkalodner @sgoldfed

Reply:
I opted to keep this constant with the "developer quickstart" section. We can update this in a subsequent PR